services/backblaze.yml

```diff
@@ -0,0 +1,8 @@
+id: backblaze
+name: Blackblaze
+rules:
+  - '@@||www.backblaze.com^'
```

Comment:
not sure about these exceptions

Reply:
@Sergey-Lyapin this allows users to visit the main website